### PR TITLE
Add Snappy package

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2128
* https://travis-ci.org/ingenue/hunter/builds/262825219?utm_source=github_status&utm_medium=notification

**Comment by chfast** [comment]
Can you explain the TOOLCHAIN=analyse failures? The log is hard to parse.

**Comment by ruslo** [comment]
`analyzer` thinks that nullpointer can be used in `memcpy`:
```
/home/travis/build/ingenue/hunter/_testing/Hunter/_Base/13efc7d/487d0d9/9e4de4b/Build/Snappy/Source/snappy.cc:1376:5: warning: Null pointer passed as an argument to a 'nonnull' parameter
    memcpy(op_ptr_, ip, avail);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by chfast** [comment]
I've seen this, but it is a warning. I'll report upstream.

**Comment by ruslo** [comment]
> I've seen this, but it is a warning

This is how clang analyzer works - it reports warnings. Polly toolchain `analyze` use script to turn it into error, otherwise this toolchain is useless :)

> I'll report upstream.

Okay, but if you're not planning to fix it fast it make sense to exclude it. We can always enable it back if needed.

**Comment by chfast** [comment]
I'd like to disable it.

The issue with memcpy() is known problem. Standard says you cannot pass null pointer there, but everyone knows implementations handles nulls anyway.

**Comment by chfast** [comment]
Can we proceed?

**Comment by ruslo** [comment]
Feel free to send pull request with exclude to [pkg.snappy](https://github.com/ingenue/hunter/tree/pkg.snappy).

**Comment by chfast** [comment]
Have you tested it?

**Comment by ruslo** [comment]
> Have you tested it?

I haven't received notification that you've updated https://github.com/ingenue/hunter/pull/110. Will run tests now.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2141
* https://travis-ci.org/ingenue/hunter/builds/263488582?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2145
* https://travis-ci.org/ingenue/hunter/builds/263969435?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.68
